packages/core/useEventListener/index.test.ts

```diff
@@ -136,16 +132,6 @@ describe('useEventListener', () => {
         listeners.forEach(listener => expect(listener).toBeCalledTimes(index + 1))
       })
     })
-
-    it('should remove all listeners with all events', () => {
```

Comment:
maybe you could have an equivalent test for an abort signal, and just assert `signal.aborted` is `true`?

i know we already test teardown by asserting calls are `0`, but its probably still worth testing the signal got aborted

Reply:
Not that easy. We have to mock `AbortSignal` to do this, no?